CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -83,5 +85,16 @@ public IList<TimeSheet> RetrieveTimeSheets(DateTimeOffset dateFrom, DateTimeOffs
                 }
             ).ToList(); 
         }
+        
+        private JObject RetrieveATimeSheet(DateTimeOffset dateFrom, DateTimeOffset dateTo, int page)
```

Comment:
Maybe rename the method to GetTimesheetFromEndpoint or something like that?

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/33/commits/8314ac0c014410c646e32e0993d8a4ec019226e2. Renamed to `RetrieveATimeSheetWithPagination`.